docs/sphinx/examples/cpp/basics/bernstein_vazirani.cpp

```diff
@@ -0,0 +1,78 @@
+#include <cudaq.h>
+#include <iostream>
+
+// If you have a NVIDIA GPU you can use this example to see
+// that the GPU-accelerated backends can easily handle a
+// larger number of qubits compared the CPU-only backend.
+
+// Depending on the available memory on your GPU, you can
+// set the number of qubits to around 30 qubits, and set the
+// `--target=nvidia` from the command line.
+
+// Note: Without setting the target to the `nvidia` backend,
+// a 30 qubit simulation simply seems to hang; that is
```

Comment:
In PR#1295, there was a discussion about the term `hang`...
Ref: https://github.com/NVIDIA/cuda-quantum/pull/1295#discussion_r1504518984